client/devdocs/style.scss

```diff
@@ -140,6 +140,31 @@
 	margin-top: 48px;
 }
 
+.design-assets {
+	@include breakpoint( "<660px" ) {
+		padding: 0 6px;
+	}
+}
+
+.design-assets h2,
```

Comment:
any chance we can scope these better while we're in here? they make styling components that want to use these tags a pain.